utils/check-for-deprecations.js

```diff
@@ -0,0 +1,199 @@
+const basePath = `https://staging-insights-api.newrelic.com/v1/accounts/${ACCOUNT_ID}/query?nrql=`;
+const options = {
+  headers: {
+    "X-Query-Key": queryKey,
+  },
+};
+
+const context = github.context;
+const octokit = new Octokit({
+  auth: process.env.GITHUB_TOKEN,
+});
+
+/**
+ * Stringifies an array for use in a NRQL query
+ * @param {Array} arr - An array of appName strings
+ * @returns {String} stringifiedArray
+ */
+const stringifyArrayForNrql = (arr) => {
+  const stringifiedArray = Object.values(arr).reduce((acc, name) => {
+    acc += ",";
+    return (acc += `'${name}'`);
+  }, "");
+
+  return stringifiedArray;
+};
+
+/**
+ * Fetches event data from the Insights Event API
+ * @param {String} url - The url to send the request to
+ * @param {Object} options - The request options
+ * @returns {Object} response.json()
+ */
+const getData = async (url, options) => {
+  try {
+    const response = await fetch(url, options);
+
+    if (response.status >= 300) {
+      throw new pRetry.AbortError({
+        errorMessage: `Request failed after ${retries} retries`,
+        statusText: response.statusText,
+        status: response.status,
+        retries,
+        url,
+      });
+    }
+    return response.json();
+  } catch (err) {
+    core.setFailed("Action failed with error:");
+    core.error(JSON.stringify(err));
+    return [];
+  }
+};
+
+/**
+ * Replaces %s in messages with an array of arguments
+ * @param {Object} events - Events from API containing message and args
+ * @returns {Array} An array of unique messages with args replaced
+ */
+const replaceMessageArgs = async (events) => {
+  const replacedMessages = await Promise.all(
+    events.map(async (event) => {
+      const args = JSON.parse(event.args);
+
+      let message = await args.reduce((acc, arg) => {
+        acc = acc.replace(/%s/, arg);
+        return acc;
+      }, event.message);
+
+      return message;
+    })
+  );
+
+  const uniqueMessages = await getUniqueMessages(replacedMessages);
+  return uniqueMessages;
+};
+
+/**
+ * Removes duplicates from an array
+ * @param {Array} messages - Messages with args replaced
+ * @returns {Array} An array of Unique Messages
+ */
+const getUniqueMessages = async (messages) => {
+  const uniqueMessages = await Promise.all(
+    messages.reduce((acc, current) => {
+      const dupe = acc.find((item) => item === current);
+
+      if (!dupe) {
+        return acc.concat([current]);
+      } else {
+        return acc;
+      }
+    }, [])
+  );
+
+  return uniqueMessages;
+};
+
+/**
+ * Fetches event data from the Insights Event API
+ * @param {Array} data - An array of objects containing appNames and messages
+ * @returns {String} pullRequestDescription
+ */
+const createPullRequestDescription = (data) => {
+  const pullRequestDescription = data.reduce((acc, app) => {
+    acc += `\n - [ ] ${app.appName}\n`;
+    app.messages.forEach((message) => (acc += `      - [ ] ${message}\n`));
```

Comment:
```suggestion
    app.messages.forEach((message) => (acc += `\t- [ ] ${message}\n`));
```
Not required, but a `\t` should get you a tab character